spec/initializers/rack/attack_spec.rb

```diff
@@ -121,11 +121,31 @@
           post messages_path, params: { message: new_message }, headers: headers
         end
         throttled_response = post messages_path, params: { message: new_message }, headers: headers
-        new_api_response = post messages_path, params: { message: new_message }, headers: dif_headers
+        new_response = post messages_path, params: { message: new_message }, headers: dif_headers
 
         valid_responses.each { |r| expect(r).not_to eq(429) }
         expect(throttled_response).to eq(429)
-        expect(new_api_response).not_to eq(429)
+        expect(new_response).not_to eq(429)
+        expect(Honeycomb).to have_received(:add_field).with("fastly_client_ip", "5.6.7.8").exactly(6).times
+        expect(Honeycomb).to have_received(:add_field).with("fastly_client_ip", "1.1.1.1").exactly(2).times
+      end
+    end
+
+    xit "throttles viewing tags" do
```

Comment:
I added this test but the problem is the endpoint does not respond fast enough to trigger the throttling. Even though we have Timecop.freeze, that does not stop Redis's internal clock and therefore we can't easily test this endpoint. Beyond redefining the controller endpoint I was not sure the best way to handle stubbing it. Suggestions?
```ruby
allow_any_instance_of(StoriesController).to receive(:index) { "" }

# causes

ActionController::MissingExactTemplate:
       StoriesController#index is missing a template for request formats: text/html
```